src/Discovery/DiscoveryDropdownTagViewer.tsx

```diff
@@ -122,7 +124,34 @@ const DiscoveryDropdownTagViewer: React.FunctionComponent<DiscoveryTagViewerProp
               xl={12}
               xxl={12}
             >
-              { tags }
+              <table >
+                <tr>
+                  <td >
+                  <Col >
+                  <Tooltip
+                  placement='left'
+                  overlay={category.tooltip}
```

Comment:
the `tooltip` field should be added to the docs at `docs/portal_config.md`

Reply:
I added  tooltip": "additional data", // optional string to add data when hovered on the info button at  portal_config.md